docs/tutorials/projects/filtered-camera-module.md

```diff
@@ -154,9 +154,10 @@ Once you have created a dataset containing the images you want to use, draw boun
    Ensure that at least 80% of the images in your dataset are labelled, with the remaining images not containing objects to identify.
    If you want your machine to be able to identify multiple objects, you can add multiple labels per image as well.
 
-### Train an ML model
+### Train a new ML model
```

Comment:
I think we want to make this more general

"Add a new model" or "Train or upload a new model"

Reply:
Right on. I've now done I think a better job clarifying flow here: one, or the other. This time with info alerts and callbacks. LMK what you think.